provider/cluster_rosa_classic_resource.go

```diff
@@ -645,9 +660,36 @@ func createClassicClusterObject(ctx context.Context,
 	return object, err
 }
 
-func (r *ClusterRosaClassicResource) validateAccountRoles(ctx context.Context, state *ClusterRosaClassicState) error {
-	r.logger.Debug(ctx, "Validating if cluster version is compatible to account roles' version")
-	region := state.CloudRegion.Value
+func validateHttpTokens(ctx context.Context, logger logging.Logger, state *ClusterRosaClassicState, version string) error {
```

Comment:
This function can also be in `common` - if it is shared with the tests

Reply:
Why in common? I mean it is not something common to share and relevant only here, so idk